OxfordBodleian/Juel-Jensen/BDLaethe39.xml

```diff
@@ -65,189 +61,789 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                        </msItem>
+                            <msItem xml:id="p1_i4">
+                            <locus from="87va" to="87vb"/>
+                                <title type="complete">Enumeration of the Old and New Testament books</title>                        
+                            <textLang mainLang="gez"/>  
+                            <!--this is the complete text-->
+                            <incipit xml:lang="gez">
+                                <locus target="#87va"/>
+                                ኈልቈ፡ መጻሕፍት፡ ዘብሉይ፡ ወዘሐዲስ፡ ፰ ኦሪት፡ ፲ወ፬ ደቂቀ፡ ነቢያት፡ ፭ ጦቢት፡ ፬ ሕፁፃን፡ ኢዮብ፡ እዝራ፡ ካልእ፡ እዝራ፡ ዳዊት፡ ሰሎሞን፡
+                                ፬ሲራክ፡ መቀብያን፡ ዮዲት፡ አስቴር፡ ነገሥት፡ ኢሳይያስ፡ ኤርምያስ፡ ዳንኤል፡ ሕዝቅኤል፡ ፵ወ፮ ብሉይ። <cb n="b"/>
+                                ወሐዲስኒ፡ ፬ወንጌል፡ ፫ ሲኖዶስ፡ ፲፬ ጳውሎስ፡ ፯ ሐዋርያት፡ ግብር፡ ቀለምሲስ፡ ፴፭ ሐዲስ።
+                            </incipit>
+                        </msItem>  
+                            <msItem xml:id="p1_i5">
+                            <locus target="#87vb"/>
+                                <title type="complete" ref="LIT4099MashafaT">Interpretation of the seven alphabetical orders of the Ethiopic script</title>
+                            <!-- A presumably longer version is found in BLadd16223#ms_i4 and BLadd16204#ms_i2-->
+                            <textLang mainLang="gez"/>    
+                            <!--this is the complete text-->
+                            <incipit xml:lang="gez">
+                                <locus target="#87va"/>
+                                ትርጓሜ፡ ቅዱስ፡ ፊደል፡ ግዕዝ፡ ወራብዕ፡ አብ፡ <sic resp="MV">ካዕበ፡</sic> ወኀምስ፡ ወልድ፡ ሣልስ፡ ወሳድስ፡ መንፈስ<supplied reason="omitted">፡</supplied> ቅዱስ፡
+                                ሳብእ፡ ፩ አምላክ። ካልእ፡ ትርጓሜ፡ ግዕዝ፡ አብ፡ ካዕበ፡ ወልድ፡ ሣልስ፡ መንፈስ<supplied reason="omitted">፡</supplied> ቅዱስ፡ ራብዕ፡ ቤተ፡ ክርስቲያን፡ 
+                                ኃምስ፡ ጥምቀት፡ ሳድስ፡ ምስሐ፡ ደብረ፡ ጽዮን፡ ሳብእ፡ ትንሣኤ፡ ሙታን።
+                            </incipit>                           
+                        </msItem> 
+                    </msContents>
                     
+                        <physDesc>
+                            <objectDesc form="Codex">
+                                <supportDesc>
+                                    <support>
+                                        <material key="parchment"/>                                     
+                                        <p>Holes, presumably resulting from parchment making, are found on e.g. <locus target="#1 #36 #52 #83"/>.
+                                            A tear has been repaired on <locus target="#63"/>
+                                        </p>
+                                    </support>
+                                    <foliation>Pencil foliation in the upper right corner carried out by the Bodleian Library during our stay in January 2020.</foliation>
+                                    <collation>
+                                        <list>
```

Comment:
Yes, perhaps the information looks redundant if appears so written in the printed catalogue. Also, some of the pieces of information might be moved under "Condition" since they concern the current state of preservation of the quires rather than their structure.
E.g. I added the following sentences inside Condition: `A protective strip of textile is sewn around <ref target="#q5 #q7 #q9 #q11 #q12 #q13 #q14 #q17 #q18"/>.
                                        A parchment support has been added around <ref target="#q1"/>.
                                        Many leaves from <ref target="#q5 #q7 #q9 #q11 #q13"/> are detached and sewn together.`.
I would wait for a revision of the material part before removing all information from the notes in the quire structure.

Reply:
Maybe there is some misunderstanding, due to the location of my comment. I was referring to the quire structure, not foliation. The update inside condition is great, then we see how to proceed when Eliana has worked on this.
Regarding foliation, I would say that "during our stay" can be omitted, but I would be in favour of keeping the information on the date of foliation in each single record because a) some manuscripts were already foliated and b) many people ignore catalogue introductions and only read single descriptions, and foliation information is absolutely crucial to the history of the manuscript and its research.

Reply:
Thinking also to the online database (where single MS records will be visualized by users), it is fine to me to keep the sentence on foliation. Then, if we deem this sentence redundant, we might still decide to leave this information out during the XML-to-PDF transformation.
I removed "during our stay" from this record and I will try to remember to do so in the next descriptions. Yet, since this sentence is found in all descriptions, we might take note of this systematic change for a later stage in which a general revision of all records will take place.

Reply:
I also think it's can be valuable to record foliation history. As Thea has stated only some mss required foliation. In the intro we could mention that this was done by Cesar.

Reply:
Ah then yes, sorry, I misunderstood and I have note understood to what you were referring to @thea-m!